punchbowl/data/tests/test_wcs.py

```diff
@@ -80,115 +80,246 @@ def test_extract_crota_helio():
     wcs_helio.wcs.pc = pc
     assert np.allclose(extract_crota_from_wcs(wcs_helio), 10 * u.deg)
 
-def test_wcs_many_point_2d_check():
-    m = NormalizedMetadata.load_template("CTM", "2")
-    date_obs = Time("2024-01-01T00:00:00", format='isot', scale='utc')
-    m['DATE-OBS'] = str(date_obs)
+def test_helio_to_celestial_wcs_many_points():
+    for date_obs, crval1, crval2, crot in product(
+            ["2021-03-20T00:00:00", "2021-01-20T00:00:00"],
+            [-20, 10, 30],
+            [-10, 0, 10],
+            [-30, 0, 30] * u.deg):
```

Comment:
Looping over permutations like this could be done with `pytest.mark.parametrize`, but then 54 separate cases would show up in the test output!